dotnet/samples/Concepts/ChatCompletion/AzureAIInference_ChatCompletion.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.AzureAIInference;
+
+namespace ChatCompletion;
+
+// The following example shows how to use Semantic Kernel with Azure AI Inference / Azure AI Studio
+public class AzureAIInference_ChatCompletion(ITestOutputHelper output) : BaseTest(output)
+{
+    [Fact]
+    public async Task ServicePromptAsync()
+    {
+        Console.WriteLine("======== Azure AI Inference - Chat Completion ========");
+
+        var chatService = new AzureAIInferenceChatCompletionService(
+            endpoint: new Uri(TestConfiguration.AzureAIInference.Endpoint),
+            apiKey: TestConfiguration.AzureAIInference.ApiKey);
+
+        Console.WriteLine("Chat content:");
+        Console.WriteLine("------------------------");
+
+        var chatHistory = new ChatHistory("You are a librarian, expert about books");
+
+        // First user message
+        chatHistory.AddUserMessage("Hi, I'm looking for book suggestions");
+        await MessageOutputAsync(chatHistory);
+
+        // First bot assistant message
+        var reply = await chatService.GetChatMessageContentAsync(chatHistory);
+        chatHistory.Add(reply);
+        await MessageOutputAsync(chatHistory);
+
+        // Second user message
+        chatHistory.AddUserMessage("I love history and philosophy, I'd like to learn something new about Greece, any suggestion");
+        await MessageOutputAsync(chatHistory);
+
+        // Second bot assistant message
```

Comment:
I believe this comment refers to second message of a bot and not to a message of the second bot? Should we use assistance to be aligned with the role in output?